include/CLI/ConfigFwd.hpp

```diff
@@ -69,14 +69,10 @@ class Config {
     /// Convert a configuration into an app
     virtual std::vector<ConfigItem> from_config(std::istream &) const = 0;
 
-    /// Convert a flag to a bool representation
+    /// get a flag value
```

Comment:
First char caps for docstrings and for comments alone on a line. Comments that follow code on the same line are not capitalized. Okay for now, but slightly easier if I don't have to correct it later.